pages/index.tsx

```diff
@@ -60,13 +60,55 @@ interface SignedInProps {
   signOut: Auth['signOut']
 }
 
-function SignedIn({ session, signOut, csrfToken }: SignedInProps) {
-  const githubLinked = session.user?.linkedAccounts?.['github']
-  const twitterLinked = session.user?.linkedAccounts?.['twitter']
+function SocialAttestationProvider({name, schema, linked, description, connectUrl, csrfToken, buttonLabel, session, attestMutation}) {
```

Comment:
Would be good to define the `Props` type on this component. Something like:

```ts
interface SocialAttestationProviderProps {
   name: string
   schema: string
   ...
}
```

Then change the component signature to:

```typescript
function SocialAttestationProvider({name, schema, linked, description, connectUrl, csrfToken, buttonLabel, session, attestMutation}: SocialAttestationProviderProps)
```

This might seem like an unnecessary step, but constraining types greatly helps with maintenance